source/upgrade/notify-admin.rst

```diff
@@ -0,0 +1,25 @@
+Notify admin
+============
+
+In some cases there are Mattermost features that are limited to specific plans. When using Mattermost, users who come across these unavailable features can send a notification to System Admins indicating that they'd like access to this feature.
+
+If access to a feature requires a plan upgrade, System Admins receive notifications of these requests in order to collect data before upgrading. For example, if only one end-user requests access to AD/LDAP, it probably isn't necessary to upgrade. However, if 20 users, including Team and Channel Admins request it, you may want to consider upgrading to support this need.
+
+This feature is designed for informational purposes only, and no action is required unless you want to take action. Your Mattermost instance's functionality is not affected if you choose not to upgrade.
```

Comment:
```suggestion
This feature is designed for informational purposes only, and no action is required unless the System Admin wants to take action. The Mattermost workspace isn't affected if the System Admin chooses not to upgrade.
```

Reply:
This feature is for System Admins only - end users won't see any of this other than the "notify admin" CTA which has clear explanatory microcopy. Hence "you". What are your thoughts around leaving it as-is given that context?